include/enum_quda_fortran.h

```diff
@@ -280,6 +281,8 @@
 #define QUDA_DOMAIN_WALL_4DPC_DIRAC 7
 #define QUDA_MOBIUS_DOMAIN_WALL_DIRAC 8
 #define QUDA_MOBIUS_DOMAIN_WALLPC_DIRAC 9
+#define QUDA_MOBIUS_DOMAIN_WALL_EOFA_DIRAC 23
```

Comment:
This is a PITA `enum` thing, but based on the ordering in `enum_quda.h`, this should be 10, `QUDA_MOBIUS_DOMAIN_WALLPC_EOFA_DIRAC` becomes 11, `QUDA_STAGGERED_DIRAC` becomes 12...

This is another motivator to get this file auto-generated from `enum_quda.h`.

Reply:
I have zero knowledge of fortran ...

Reply:
@weinbe2 Let me know if this has been fixed now.